plugins/inputs/ntpq/README.md

```diff
@@ -45,11 +49,15 @@ server (RMS of difference of multiple time samples, milliseconds);
 
 ### Tags
 
-- All measurements have the following tags:
-  - refid
-  - remote
-  - type
-  - stratum
+All measurements have the following tags:
+
+- refid
+- remote
+- type
+- stratum
+
+In case you are specifying `servers`, the measurement has an
+additional `server` tag.
```

Comment:
I think we are going to chat about this in planning, but I think we want to standardize on source per #4413

Reply:
Agreed and fixed.